Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM php:7.2-cli AS dev
 
 COPY --from=composer /usr/bin/composer /usr/bin/
 
-RUN yes | pecl install xdebug \
+RUN yes | pecl install xdebug-2.9.8 \
```

Comment:
Is there a driving reason to specify a specific version here?

Reply:
Yes, without the version it will install the latest XDebug 3.2.x, which isn't compatible with PHP 7.x

2.9.8 is the last version backward compatible with PHP 7.1.